### ISSUE TITLE
pre-commit terraform_tflint doesn't work locally

### ISSUE THREAD

**Issue by max-rocket-internet** [terraform-aws-modules/terraform-aws-eks#721]
# I have issues

## I'm submitting a...

* [x] bug report
* [ ] feature request
* [ ] support request - read the [FAQ](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/faq.md) first!
* [ ] kudos, thank you, warm fuzzy

## What is the current behavior?

Same for @barryib.

Hopefully @antonbabenko know what's up.

On a Mac in bash shell:

```
$ pre-commit run --all-files terraform_tflint
Terraform validate with tflint...........................................Failed
hookid: terraform_tflint

/Users/max.williams/.cache/pre-commit/repotsUjgI/terraform_tflint.sh: line 42: pushd: modules/node_groups/node_groups.tf examples/basic/main.tf examples/managed_node_groups/variables.tf examples/launch_templates/variables.tf examples/managed_node_groups/outputs.tf local.tf modules/node_groups: No such file or directory
/Users/max.williams/.cache/pre-commit/repotsUjgI/terraform_tflint.sh: line 42: pushd: examples/managed_node_groups/main.tf modules/node_groups/locals.tf cluster.tf irsa.tf examples/launch_templates/outputs.tf examples/basic/variables.tf examples/spot_instances: No such file or directory
/Users/max.williams/.cache/pre-commit/repotsUjgI/terraform_tflint.sh: line 42: pushd: workers_launch_template.tf examples/spot_instances/variables.tf data.tf variables.tf node_groups.tf examples/basic/outputs.tf examples/spot_instances: No such file or directory
/Users/max.williams/.cache/pre-commit/repotsUjgI/terraform_tflint.sh: line 42: pushd: modules/node_groups/variables.tf modules/node_groups/random.tf examples/launch_templates: No such file or directory
```

**Comment by barryib** [comment]
I were

**Comment by barryib** [comment]
Sorry wrong button. This PR will fix the problem https://github.com/antonbabenko/pre-commit-terraform/pull/77

**Comment by antonbabenko** [comment]
Thanks for pointing to the issue and the solution.

I have just released [v1.25.0](https://github.com/antonbabenko/pre-commit-terraform/releases/v1.25.0) of `pre-commit-terraform`. Please update this repo and let me know if there is anything else pending review/merge.

**Comment by barryib** [comment]
It works ! The new release solved the problem.
